app/controllers/markups_controller.rb

```diff
@@ -0,0 +1,13 @@
+class MarkupsController < AuthenticatedController
```

Comment:
- This is not a "collection" resources controller, seems like a singular `Markup` would be a better fit.

- if all actions of the controller are using `layout: false` maybe it should be a controller setting.